src/Actions/InviteAction.php

```diff
@@ -43,8 +45,10 @@ protected function setUp(): void
                 // Use the method if the developer has specified one
                 if (method_exists($user, 'sendPasswordSetNotification')) {
                     $user->sendPasswordSetNotification($token);
+                } elseif (method_exists($user, 'notify')) {
+                    $user->notify(new SetPassword($token, Filament::getId()));
```

Comment:
We can use `Filament::getCurrentPanel()?->getId()` instead of `Filament::getId()` for better panel context:

```suggestion
                    $user->notify(new SetPassword($token, Filament::getCurrentPanel()?->getId()));
```